packages/extension/src/localStore/tests/updateStorage.ts

```diff
@@ -25,6 +25,11 @@ describe('local store: updateStorage', () => {
 
   beforeAll(() => {
     globalThis.chrome = {
+      //@ts-ignore local does not implementations of other properties
+      action: {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        setBadgeText: async () => {},
```

Comment:
Create `noop` util?

Reply:
It's not used anywhere else. We can create it when that happens